### PR TITLE
Blog: November updates

### PR DESCRIPTION
**Pull request by psiinon**
Screenshot 2022-12-01 at 13-52-27 ZAP Updates 2022 November

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/blog/2022-12-01-zap-updates-2022-november/index.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+It seemed to go down well and I'll share the recording once it has been published.
+
+I talked to a lot of people and heard some great feedback about ZAP!
+I also made some very good contacts, including people and companies who we now plan to collaborate with - 
+more details when we have something to show :wink:.
+
+### DevOpsDays Tel Aviv
+
+I will be giving a Spotlight Talk at [DevOpsDays Tel Aviv](https://devopsdays.org/events/2022-tel-aviv/welcome/) on 
+"Evolution of OSS: Perspective from a Decade of Maintaining A (Wildly) Popular Open Source Project "  - no prizes for guessing which project that is :wink:.
```
Comment (kingthorin) [inline site/content/blog/2022-12-01-zap-updates-2022-november/index.md:57]:
```suggestion
"Evolution of OSS: Perspective from a Decade of Maintaining A (Wildly) Popular Open Source Project " - no prizes for guessing which project that is :wink:.
```

**Comment by kingthorin** [comment]
Looks good overall.

File: site/content/blog/2022-12-01-zap-updates-2022-november/index.md (hunk 1 above)
Comment (psiinon) [reply site/content/blog/2022-12-01-zap-updates-2022-november/index.md]: Done :)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by ricekot — APPROVED** [review]
Thanks!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**